switch_model/wecc/get_inputs/get_inputs.py

```
@@ -9,33 +9,26 @@
 """
 
 # Standard packages
-import argparse
-import os
-import shutil
+import warnings
 from typing import Iterable, List
 
 # Switch packages
-from switch_model.utilities import query_yes_no, StepTimer
-from switch_model.wecc.utilities import load_config, connect
-
-# Third-party packages
-import pandas as pd
-
+from switch_model.wecc.utilities import connect
 
 def write_csv_from_query(cursor, fname: str, headers: List[str], query: str):
     """Create CSV file from cursor."""
-    print(f"{fname}.csv... ", flush=True)
+    print(f"\t{fname}.csv... ", flush=True)
     cursor.execute(query)
     data = cursor.fetchall()
     write_csv(data, fname, headers, log=False)
     if not data:
-        print("Warning: File is empty.")
+        warnings.warn(f"File {fname} is empty.")
```

I think we should raise an error instead if the data is empty. By doing so, we will identify if there is an error on the configuration file, instead of waiting for it to finish.

---

I think in some scenarios we may want the file to be empty though. Like e.g. if we're working in a single zone the transmission lines files is empty. In the end I don't think this matters much, either way the user is warned about a potential problem.